plugins/carbonetes-scan.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: carbonetes-scan
+spec:
+  homepage: https://github.com/carbonetes/kubectl-carbonetes-scan
+  shortDescription: Seamlessly integrates complete container analysis directly into your Kubernetes cluster.
```

Comment:
shortening by half would be ideal, lots of redundant/subjective words in there.

Reply:
Thanks for the review. We'll make a change to this description.